### ISSUE TITLE
Differences from the official Tensorflow implementation

### ISSUE THREAD

**Issue by yida1996** [justanhduc/brushstroke-parameterized-style-transfer#3]
Great job reimplementing in pytorch!  However, In the case of the same number of iterations(1000), I found the result different from the official implementation(And I see the parameters used in your code are the same as the official version). Are there any changes in your code that affect the final result?  Images are as follows.

Looking forward to your reply.

offical 1000 iterations
![offical_1000iter](https://user-images.githubusercontent.com/49750073/120293978-547ba780-c2f8-11eb-9766-6983eb323a4a.jpg)
pytorch 1000 iterations
![pytorch_1000iter](https://user-images.githubusercontent.com/49750073/120293990-56de0180-c2f8-11eb-882c-e9a20b6e360e.jpg)

**Comment by justanhduc** [comment]
Hey @yida1996. Yes this is expected. The main difference is the pretrained VGG I used doesn't come from the official repo. Also, the layers used for the content and style losses are different too. You may observe that the loss values between this implementation and the official one are different. However, if you compare the results after the brush stroke optimization, you can see they are similar. To make the two the same, you can either tune the loss weights or convert the VGG in the official repo to Pytorch.
